### DISCUSSION TITLE
'GrouperView' object has no attribute 'join'

### DISCUSSION THREAD

**Discussion by rootsmusic** [mwaskom/seaborn#3521] [Q&A] [answered]
My environment has installed seaborn 0.11.2 (preferred version).  I'm running the simple script below.
```
import seaborn as sns
import pandas as pd
import numpy as np

# Creating a sample DataFrame
data = {
    'A': np.random.randn(100),
    'B': np.random.randn(100),
    'C': np.random.randn(100),
    'D': np.random.randn(100)
}
df = pd.DataFrame(data)

# Create a pair plot
sns.pairplot(df)
```
Why does the last line output the error: AttributeError: 'GrouperView' object has no attribute 'join'?

**Comment by mwaskom** [comment, marked as answer]
I'm not sure what you mean by "preferred version" but 0.11.2 was released more than two years ago and there have been two major release cycles since. I cannot replicate your issue with the latest version (0.13.0). If for some reason you are unable to upgrade your seaborn, you may need to downgrade seaborn dependencies to contemporary versions.
